clang/test/CodeGenHIP/dpp-const-fold.hip

```diff
@@ -0,0 +1,48 @@
+// REQUIRES: amdgpu-registered-target
+
+// RUN: %clang --offload-arch=gfx906 -S -o - -emit-llvm --cuda-device-only -nogpuinc -nogpulib\
```

Comment:
This should use %clang_cc1, look at some of the other CodeGen* test run lines for examples